src/components/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import styles from './CheckboxGroup.st.css';
+import { Checkbox } from '../Checkbox';
+
+export enum Layout {
```

Comment:
The Layout enum is clashing with the enum from Ratings.
We need to change its name

Reply:
Re-exported explicitly